.github/workflows/publish-docs.yml

```diff
@@ -12,6 +12,8 @@ on:
 
 jobs:
   build:
+    # only build and deploy docs if the actor is not dependabot
+    if: ${{ github.actor != 'dependabot[bot]' }}
```

Comment:
What does actor mean in this context? What would be the actor if it isn't dependabot?

Reply:
Thanks for this question - the actor for GitHub actions is the latest person to trigger a workflow. That could be one of us or also automated agents like Dependabot, which act on our / GitHub's behalf to perform work.